ConsentViewController/Classes/SPError.swift

```diff
@@ -75,6 +75,23 @@ import Foundation
     required init?(coder aDecoder: NSCoder) { fatalError("init(coder:) has not been implemented") }
 }
 
+@objcMembers public class LogCustomMetricsChildPmIdError: SPError {
```

Comment:
@Nevazhnovu @carmelo-iriti are we using the same `spCode` in both SDKs?

@Nevazhnovu can I suggest naming the class `MissingChildPmIdError`?

Reply:
1) 99% that code is different
2) no problem!